packages/manager/apps/telecom/src/app/telecom/telephony/carrierSip/dashboard/cdr/cdr.routing.js

```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.telephony.carrierSip.cdr', {
+    url: '/cdr',
+    views: {
+      '@telecom.telephony.carrierSip': 'carrierSipCdr',
+    },
+    resolve: {
+      billingAccount: /* @ngInject */ $transition$ => $transition$.params().billingAccount,
```

Comment:
`resolve` already declared in parent state

Reply:
It's done.